### PR TITLE
HAWQ-196: Refine GUCs in hawq-site.xml and template-hawq-site.xml

### PR DESCRIPTION
**Pull request by huor**
Please review the unification of GUCs in hawq-site.xml and template-hawq-site.xml

### CODE REVIEW

**Comment by huor** [comment]
@radarwave , @jiny2 , @yaoj2, @changleicn , please review.

File: src/backend/utils/misc/etc/template-hawq-site.xml [hunk 1]
```diff
@@ -20,131 +20,133 @@ under the License.
...
+	<property>
+		<name>hawq_segment_temp_directory</name>
+		<value>%segment.temp.directory%</value>
+		<description>The temporary directory reserved for hawq segment.</description>
+	</property>
      
-    <!-- HAWQ resource manager parameters -->
-    <property>
-        <name>hawq_rm_yarn_address</name>
-        <value>%master.host%:9980</value>
```
Comment (radarwave) [inline src/backend/utils/misc/etc/template-hawq-site.xml]: The port number better to be consist between hawq-site.xml, template-hawq-site.xml and yarn-client.xml.

8032 <--> 9080
8030 <--> 9081

Comment (huor) [inline src/backend/utils/misc/etc/template-hawq-site.xml]: To make the ports consistent in hawq-site.xml, template-hawq-site.xml and yarn-client.xml, will use below settings:

hawq_rm_yarn_address: %master.host%:8032
hawq_rm_yarn_scheduler_address: %master.host%:8030

**Comment by radarwave** [comment]
Looks good to me. +1

File: src/backend/utils/misc/etc/hawq-site.xml [hunk 2]
```diff
@@ -20,103 +20,133 @@ under the License.
...
+		<name>hawq_re_cgroup_mount_point</name>
+		<value>/sys/fs/cgroup</value>
+		<description>The mount point of CGroup file system for resource enforcement.
+					 For example, /sys/fs/cgroup/cpu/hawq for CPU sub-system.
+		</description>
+	</property>
+
+	<property>
+		<name>hawq_re_cgroup_hierarchy_name</name>
+		<value>hawq</value>
```
Comment (denalex) [inline src/backend/utils/misc/etc/hawq-site.xml]: Ambari currently sets this value as hadoop-yarn: https://github.com/Pivotal-Hadoop/ambari/blob/trunk/ambari-server/src/main/resources/common-services/HAWQ/2.0.0/configuration/hawq-site.xml#L169

Is this a problem ? Which value should be used ?

Comment (huor) [inline src/backend/utils/misc/etc/hawq-site.xml]: The previous value for "hawq_re_cgroup_hierarchy_name" is "hadoop-yarn". In the unification of properties, we changed it to "hawq" which is more meaningful. Please update it to "hawq" in Ambari accordingly. Thanks.

**Pull request closed without merging**